frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/SimpleGraphTable.java

```diff
@@ -13,30 +13,26 @@
 import com.virtuslab.gitmachete.frontend.ui.impl.cell.BranchOrCommitCell;
 import com.virtuslab.gitmachete.frontend.ui.impl.cell.BranchOrCommitCellRendererComponent;
 
-public final class DemoGraphTable extends JBTable implements IGitMacheteRepositorySnapshotProvider {
-
-  private static final GraphTableModel GRAPH_TABLE_MODEL = deriveGraphTableModel();
-
+public final class SimpleGraphTable extends JBTable implements IGitMacheteRepositorySnapshotProvider {
```

Comment:
Hmm I don't like the current distinction: `Simple` vs `GitMachete` graph table :thinking:

Reply:
As agreed via Slack `GitMacheteGraphTable` will be renamed to `EnhancedGraphTable` + javadoc to explain the different between this two classes